### DISCUSSION TITLE
Todoist Project Notes Plugin

### DISCUSSION THREAD

**Discussion by davidmarttila** [jamiebrynes7/obsidian-todoist-plugin#290] [Show and tell]
Hello! Thanks for developing this plugin! I've made a small plugin for myself that is meant to work together with Todoist Sync. It creates project notes that mirror your Todoist Project structure, and supports templating so that you can automatically insert Todoist Sync commands showing relevant tasks to the project. If you're interested, it's available here: https://github.com/dsuedholt/obsidian-todoist-project-notes

**Comment by jamiebrynes7** [comment]
This is really cool @dsuedholt! I'd be keen to collaborate in the future for making the integration tighter, for example - exposing an API client for you to use (so folks don't need to input their API token twice) or even potentially bringing this functionality into this plugin directly?

Let know what you think!
